src/puppetlabs/ezbake/dependency_utils.clj

```diff
@@ -214,3 +214,7 @@
                     out-dir-fn
                     lein-project)
            deps))))
+
+(defn snapshot-version?
+  [version]
+  (boolean (re-find #"SNAPSHOT" version)))
```

Comment:
pomegranate uses a slightly more strict regex (includes a `-`), so maybe we should add a `-`? (https://github.com/cemerick/pomegranate/blob/e44f45fa3abc29dbc0ab88f98b9bbbada7819128/src/main/clojure/cemerick/pomegranate/aether.clj#L831). Probably not a big deal either way.